debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -1157,6 +1157,57 @@ public void shouldAllowForCustomSnapshot() throws InterruptedException {
         VerifyRecord.isValidRead(s2recs.get(1), PK_FIELD, 2);
     }
 
+    @Test
```

Comment:
Could you add the `@FixFor` annotation (here and elsewhere)?

Reply:
Sure, Will add them across all tests.